apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -0,0 +1,194 @@
+/** Workshop Survey page. Displays survey results. ...*/
+import $ from 'jquery';
+import _ from 'lodash';
+import React from 'react';
+import {
+  Row,
+  Col,
+  Button
+} from 'react-bootstrap';
+import moment from 'moment';
+import 'react-select/dist/react-select.css';
+import {DATE_FORMAT} from '../workshopConstants';
+
+const rowOrder = [
+  {text: 'Number of attending teachers', key: 'number_teachers'},
+  {text: 'Number of survey responses', key: 'response_count'},
+  {text: 'Facilitator Effectiveness (out of 5)', key: 'facilitator_effectiveness'},
+  {text: 'Overall Success Score (out of 6)', key: 'overall_success'},
+];
+
+const SurveyResultsHeader = React.createClass({
+  contextTypes: {
+    router: React.PropTypes.object.isRequired
+  },
+
+  propTypes: {
+    workshops: React.PropTypes.arrayOf(
+      React.PropTypes.shape({
+        id: React.PropTypes.number.isRequired,
+        sessions: React.PropTypes.array.isRequired,
+        location_name: React.PropTypes.string.isRequired,
+        workshop_type: React.PropTypes.string.isRequired,
+        course: React.PropTypes.string.isRequired,
+        enrolled_teacher_count: React.PropTypes.number.isRequired,
+        capacity: React.PropTypes.number.isRequired,
+        facilitators: React.PropTypes.array.isRequired,
+        organizer: React.PropTypes.shape({
+          name: React.PropTypes.string.isRequired,
+          email: React.PropTypes.string.isRequired
+        }).isRequired
+      })
+    )
+  },
+
+  getInitialState() {
+    return {
+      course: '',
```

Comment:
I think this was my fault from an earlier conversation, but this should match the key in setState. I'd suggest changing this to `selectedCourse` but I'd also accept changing the one on line 65 to `state`.